src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -458,6 +458,10 @@ export class BrightScriptDebugSession extends BaseDebugSession {
                 });
             }
 
+            this.rokuAdapter.on('device-unresponsive', () => {
+                this.showPopupMessage('No telnet output received for a while. Consider stopping the debug session', 'warn', false, 'Stop Debugger');
```

Comment:
What about this message instead?

Roku device <1.2.3.4> is not responding. A session restart may be needed. Active command:\n
"print 123".